lib/storekit/error.rb

```diff
@@ -9,14 +9,14 @@ def initialize(status_code)
       @status_code = status_code
       message =
         case status_code
-        when 21000 then 'The App Store could not read the JSON object you provided.'
-        when 21002 then 'The data in the receipt-data property was malformed or missing.'
-        when 21003 then 'The receipt could not be authenticated.'
-        when 21004 then 'The shared secret you provided does not match the shared secret on file for your account.'
-        when 21005 then 'The receipt server is not currently available.'
-        when 21006 then 'This receipt is valid but the subscription has expired.'
-        when 21007 then 'This receipt is from the test environment, but it was sent to the production environment for verification.'
-        when 21008 then 'This receipt is from the production environment, but it was sent to the test environment for verification.'
+        when 21000 then 'The App Store could not read the JSON object you provided (21000).'
+        when 21002 then 'The data in the receipt-data property was malformed or missing (21002).'
+        when 21003 then 'The receipt could not be authenticated (21003).'
+        when 21004 then 'The shared secret you provided does not match the shared secret on file for your account (21004).'
+        when 21005 then 'The receipt server is not currently available (21005).'
+        when 21006 then 'This receipt is valid but the subscription has expired (21006).'
+        when 21007 then 'This receipt is from the test environment, but it was sent to the production environment for verification (21007).'
+        when 21008 then 'This receipt is from the production environment, but it was sent to the test environment for verification (21008).'
```

Comment:
I chose only the first sentence because it describes the error succinctly without editorializing its meaning.

The point still stands that the addition of a status code into the message string is redundant because it is already available as an attribute on the error object. If your application needs this error code to be printed with the message string, feel free to solve it in the application instead of pushing this into the gem.